07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -460,7 +460,36 @@ image below:
 This change will be effective across your organization immediately. Anyone running AzSK commands (in fresh PS sessions)
 should see the new message. 
 
-##### b) Changing a control setting for specific controls 
+##### b) Customize output folder for my org
+Normally, when we run GSS the security scan results go to a default folder 
+`%LOCALAPPDATA%\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]E.g.`
+C:\Users\UserName\AppData\Local\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]\20170331_142819. 
+This location can be customized in an org policy so that the scan results can be collected at a particular location on org policy users machine.
+
+###### Steps:
+
+i) Open the AzSk.json from your local org-policy folder. Path for example: D:\ContosoPolicies\Config\AzSK.json
+
+ii) Add new field OutputFolderPath and give an particular path where you want to store GSS/GRS security scan results 
+E.g.
+```PowerShell
+     "OutputFolderPath":  "Path"
+```
+iii) Save the file
+
+iv) Rerun the policy update or setup command (the same command you ran for the first-time setup)
+
+###### Testing: 
+
+Now anyone in your org if start a fresh PS console and run GSS/GRS cmdlet then its security scan result logs will be collected at a particular defined location in OutputFolderPath field added in azsk.json file.
```

Comment:
Now anyone in your org if starts a fresh PS console and run GSS/GRS cmdlet then its security scan result logs will be collected at the custom defined location in OutputFolderPath field that was added in azsk.json file

Now if anyone in your org starts a fresh sessions and performs the AzSK security scans, the results would be saved in the custom defined path that is present in AzSK.json file